<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoteWindow</class>
 <widget class="QMainWindow" name="VoteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    color: rgb(0, 0, 0);
    font-size: 14px;
    border-radius: 10px;
    padding-left: 15px;
    border: 2px solid rgb(191, 191, 191); /* Границы серого цвета */
    background-color: rgb(255, 255, 255);
}
</string>
      </property>
      <property name="placeholderText">
       <string>choose id</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>180</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	
	background-color: rgb(56, 82 , 151);
	border-radius:10px
}

QPushButton:hover {
	
	background-color: rgb(60, 85, 157);
}</string>
      </property>
      <property name="text">
       <string>vote</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
